alvr/session/src/settings.rs

```diff
@@ -740,6 +740,21 @@ pub struct BodyTrackingConfig {
     pub tracked: bool,
 }
 
+#[derive(SettingsSchema, Serialize, Deserialize, Clone, PartialEq)]
+pub struct VMCSinkConfig {
+    pub host: String,
+    pub port: u16,
+}
+
+#[derive(SettingsSchema, Serialize, Deserialize, Clone)]
+#[schema(collapsible)]
+pub struct VMCConfig {
+    pub sink: VMCSinkConfig,
+    #[schema(strings(help = "Turn this off to temporarily pause sending data."))]
+    #[schema(flag = "real-time")]
+    pub publish: bool,
+}
```

Comment:
Maybe these two structs can be merged? VMC is itself a sink (data output) so "VMCSinkConfig" is redundant

Reply:
Ok, TBH I was basing myself on what was already available there (BodyTrackingConfig specifically). 

I'll merge 'em and keep VMCConfig then :)